### PR TITLE
fix(spring): change @Generated description to spring specific.

### PR DESCRIPTION
**Pull request by zhumin8**
change `@Generated` description to spring specific and separate from "gapic-generator-java" for client libraries.
Also moved relevant annotation from `SpringPackageInfoComposer` to `SpringComposer`.

### CODE REVIEW

**Comment by emmileaf** [comment]
Couldn't figure out how to add suggestions to unchanged files, but this description is also set in `SpringPackageInfoComposer` (and `SpringPackageInfo.golden`). Could you help make the quick change in this PR? Thanks!

File: src/main/java/com/google/api/generator/spring/composer/SpringComposer.java [hunk 1]
```diff
@@ -81,7 +81,7 @@ protected static List<GapicClass> addExtraClassAnnotations(List<GapicClass> gapi
     AnnotationNode generatedAnnotation =
         AnnotationNode.builder()
             .setType(FIXED_TYPESTORE.get("Generated"))
-            .setDescription("by gapic-generator-java")
+            .setDescription("by spring-generator-java")
```
Comment (blakeli0) [inline src/main/java/com/google/api/generator/spring/composer/SpringComposer.java:84]: What do we want to call the spring auto config generator once we move the code to `spring-cloud-gcp`? Maybe we should come up with a long term name and stick to it, `spring-generator-java` seems a little odd to me. Maybe `spring-auto-config-generator` or just a generic `spring-generator`?

Comment (zhumin8) [reply src/main/java/com/google/api/generator/spring/composer/SpringComposer.java:84]: Thanks for the suggestions. `spring-generator` sounds better to me, to keep it open to expanding beyond auto-config in future.

Comment (meltsufin) [reply src/main/java/com/google/api/generator/spring/composer/SpringComposer.java:84]: Should we add "google" somewhere in there? For instance, `google-cloud-spring-generator`?

**Review by emmileaf — APPROVED** [review]

**Review by blakeli0 — APPROVED** [review]

**Pull request merged**